command/git.go

```diff
@@ -247,7 +263,22 @@ func (c *gitFetchProject) Execute(ctx context.Context,
 		c.Token = splitToken[1]
 	}
 
-	gitCommands, err := c.buildCloneCommand(conf)
+	opts := cloneOpts{
+		method: conf.Distro.CloneMethod,
+		owner:  conf.ProjectRef.Owner,
+		repo:   conf.ProjectRef.Repo,
+		branch: conf.ProjectRef.Branch,
+		dir:    c.Directory,
+		token:  c.Token,
+	}
+	if err = opts.setLocation(); err != nil {
+		return err
```

Comment:
wrap

Reply:
I think this is sufficiently wrapped as is